src/p_mobj.cpp

```diff
@@ -1684,7 +1684,7 @@ bool P_SeekerMissile (AActor *actor, angle_t thresh, angle_t turnMax, bool preci
 		angle_t pitch = 0;
 		if (!(actor->flags3 & (MF3_FLOORHUGGER|MF3_CEILINGHUGGER)))
 		{ // Need to seek vertically
-			double dist = MAX(1.0, FVector2(actor->Vec2To(target)).Length());
+			double dist = MAX(1.0, FVector2(target->X() - actor->X(), target->Y() - actor->Y()).Length());

```

Comment:
I honestly tried different things, but this was the only one it would really accept and I don't have much experience working with the structs themselves. At the end of the day, even if it's a step backwards, it still fixes this.